TgService/src/main/java/org/example/tgservice/patterns/HandlerTemplate.java

```diff
@@ -0,0 +1,20 @@
+package org.example.tgservice.patterns;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+public class HandlerTemplate {
+
+    private final RestTemplate restTemplate = new RestTemplate();
+
+    public ResponseEntity<?> createResponse(Object type, String action, String... variables){
```

Comment:
почему сразу не сделать?
```suggestion
    public ResponseEntity<?> createResponse(Class type, String action, String... variables){
```

а еще лучше сразу добавить дженерик
```suggestion
    public <T> ResponseEntity<T> createResponse(Class<T> type, String action, String... variables){
```

тогда тебе не придется загромождать код этим:
        ResponseEntity<String> allMessages =
                (ResponseEntity<String>) handlerTemplate.createResponse(String.class,"getAllMessages", chatId, name);
                
будет просто
        ResponseEntity<String> allMessages =
               handlerTemplate.createResponse(String.class,"getAllMessages", chatId, name);